code/datums/components/storage/storage.dm

```diff
@@ -831,13 +832,17 @@
 	return hide_from(target)
 
 /datum/component/storage/proc/on_alt_click(datum/source, mob/user)
+	to_chat(user,"<span class='warning'>This action has been moved to right click.</span>")
```

Comment:
For MSO's thing, make this call `on_right_click(source, user)` as well, while also changing this message to saying it WILL be moved off.

Eventually, we'll remove it. (lying)